app/controllers/volunteers_controller.rb

```diff
@@ -58,6 +60,11 @@ def deactivate
 
   private
 
+  def _get_available_casa_cases
```

Comment:
Why are we starting this method with an underscore? If it's to show that it's private, this just isn't common practice in Ruby, so I don't think we should do this.